src/quicktube.js

```diff
@@ -159,6 +160,50 @@ class Quicktube {
         this.videoPoster.classList.remove(this.options.posterFrameHiddenClass);
     }
 
+    onPlayerPause(videoTitle, pageTitle) {
+        console.log('pause');
+        this.removeActiveState();
+
+        if (this.options.trackAnalytics) {
+            const label = `Video Paused - ${videoTitle}`;
+            trackEvent({
+                eventCategory: 'youtube',
+                eventAction: pageTitle,
+                eventLabel: label,
+            });
+        }
+    }
+
+    onPlayerPlay(videoTitle, pageTitle, target) {
+        console.log('play');
+        this.videoEl.setAttribute('data-video-playing', true);
+        this.addActiveState();
+
+        if (this.options.trackAnalytics) {
+            // Report % played every second
+            setTimeout(this.onPlayerPercent.bind(this, target), 1000);
```

Comment:
Every second feels overly wasteful. Please use a higher time (10s?), and-or make this configurable